packages/manager/src/modules/fund-withdrawal/services/fund-withdrawal.services.ts

```diff
@@ -33,7 +26,7 @@ export const getFundWithdrawInfo = (
 
 export const withdrawFund = (id: string, onClose: () => void) => (
   value: FundWithdraw
-) => (dispatch: Dispatch) => {
+): any => (dispatch: Dispatch) => {
```

Comment:
It return Promise<any> (But I think API must return us full type. Is the backend problem?)